includes/definitions/fs-nmu.yaml

```diff
@@ -1,10 +1,19 @@
 os: fs-nmu
-text: 'Network Management Unit'
+text: 'Fiberstore Network Management Unit'
 type: network
 icon: fs
 group: fs
 mib_dir: fs
+over:
+    - { graph: device_state, text: 'Sensors' }
+    - { graph: device_temperature, text: 'Temperature' }
+    - { graph: device_dbm, text: 'dbm' }
 discovery:
     -
         sysObjectID:
             - .1.3.6.1.2.1.1.2
+            - .1.3.6.1.4.1.40989
+poller_modules:
+    sensors: true
+discovery_modules:
+    sensors: true
```

Comment:
isnt this default on? please remove in that case

Reply:
removed.

Reply:
Unfortunately due to the way FS has written their mibs, OAP-C1-OEO through OAP-C16-OEO are mibs for each of the 16 slots in the device chassis. 1.3.6.1.4.1.40989.10.16.1.2.1, 1.3.6.1.4.1.40989.10.16.2.2.1, etc. If we were to add another OEO module, or move one, to slot 9, it would need the OAP-C9-OEO mib.

Same with the OAP-C1-EDFA through OAP-C16-EDFA mibs.

Other FS customers will also have their modules populated in different slots to ours, and we believe all 16 mibs for both OEOs and EDFAs are required to cover all combinations.